test/PidTest.cpp

```diff
@@ -0,0 +1,192 @@
+/**
+ * @file PidTest.cpp
+ * @brief PidGainクラス,Pidクラスをテストする
+ * @author Negihara
+ */
+
+#include "Pid.h"
+#include <gtest/gtest.h>
+
+namespace etrobocon2023_test {
+  TEST(PidGainTest, gain)
+  {
+    double expected_p = 0.1;
+    double expected_i = 0.03;
+    double expected_d = 0.612;
+    PidGain actualPidGain(expected_p, expected_i, expected_d);
+    EXPECT_DOUBLE_EQ(expected_p, actualPidGain.kp);
+    EXPECT_DOUBLE_EQ(expected_i, actualPidGain.ki);
+    EXPECT_DOUBLE_EQ(expected_d, actualPidGain.kd);
+  }
+
+  TEST(PidGainTest, gainZero)
+  {
+    double expected_p = 0.0;
+    double expected_i = 0.0;
+    double expected_d = 0.0;
+    PidGain actualPidGain(expected_p, expected_i, expected_d);
+    EXPECT_DOUBLE_EQ(expected_p, actualPidGain.kp);
+    EXPECT_DOUBLE_EQ(expected_i, actualPidGain.ki);
+    EXPECT_DOUBLE_EQ(expected_d, actualPidGain.kd);
+  }
+
+  TEST(PidGainTest, gainMinus)
+  {
+    double expected_p = -0.5;
+    double expected_i = -0.2;
+    double expected_d = -0.3;
+    PidGain actualPidGain(expected_p, expected_i, expected_d);
+    EXPECT_DOUBLE_EQ(expected_p, actualPidGain.kp);
+    EXPECT_DOUBLE_EQ(expected_i, actualPidGain.ki);
+    EXPECT_DOUBLE_EQ(expected_d, actualPidGain.kd);
+  }
+
+  TEST(PidTest, calculatePid)
+  {
+    constexpr double DELTA = 0.01;
+    double expected_p = 0.6;
+    double expected_i = 0.02;
+    double expected_d = 0.03;
```

Comment:
どちらかというとkpかな？
PidGainのテストではその値が期待されてたからexpectedだけど今回はただの初期化用の値。
```suggestion
    double kp = 0.6;
    double ki = 0.02;
    double kd = 0.03;
```